projects/igniteui-angular/src/lib/grid/grid.component.ts

```diff
@@ -482,6 +486,10 @@ export class IgxGridComponent implements OnInit, OnDestroy, AfterContentInit, Af
         this.calcWidth = this._width && this._width.indexOf('%') === -1 ? parseInt(this._width, 10) : 0;
         this.calcHeight = 0;
         this.calcRowCheckboxWidth = 0;
+        this._subscribeOnRowAdded$ = this.onRowAdded.subscribe(() => this.clearSummaryCache());
```

Comment:
Use the `takeUntil(this.destroy$)` for unsubscribing when the grid is destroyed. Don't manage subscribtions 'manually'.